saleor/account/models.py

```diff
@@ -68,9 +68,7 @@ def __str__(self):
     def __eq__(self, other):
         return self.as_data() == other.as_data()
 
-    def __hash__(self):
-        # FIXME: in Django 2.2 this is not present if __eq__ is defined
-        return super().__hash__()
+    __hash__ = models.Model.__hash__
```

Comment:
Are there cases where `__eq__` would return `False` while `__hash__` remains `True`? Because this will break Python.